HelpSource/Classes/RLPF.schelp

```diff
@@ -42,20 +42,23 @@ Examples::
 
 code::
 
-{ RLPF.ar(Saw.ar(200, 0.1), FSinOsc.kr(XLine.kr(0.7, 300, 20), 0, 3600, 4000), 0.2) }.play;
+// don't go too fast - frequency folding will result in unexpected and LOUD filter behaviour!
+{ RLPF.ar(Saw.ar(200, 0.1), FSinOsc.kr(XLine.kr(0.7, 95, 20).poll, 0, 3600, 4000), 0.2) }.play;
```

Comment:
unless intended, `FSinOsc` is a bad choice for FM as modulator, because its amplitude is not linear.